src/fuzzfetch/fetch.py

```diff
@@ -340,7 +342,7 @@ def __init__(self, target, branch, build, flags, platform=None):
                 if self._branch is None:
                     branch = re.search(r'\.mozilla-(?P<branch>[a-z]+[0-9]*)\.', build)
                     self._branch = branch.group('branch') if branch is not None else '?'
-                asan, debug, fuzzing, coverage = self._flags
+                asan, debug, fuzzing, valgrind, coverage = self._flags
```

Comment:
`self._flags` is a `namedtuple`, so `valgrind` and `coverage` will be reversed when you unpack them out of order.